pixi.toml

```diff
@@ -144,15 +147,28 @@ pyqt-stubs = "*"
 types-requests = "*"
 qgis-plugin-manager = "*"
 libgdal-arrow-parquet = "3.7.2.*"
+juliaup = "*"
 
 [target.win-64.dependencies]
 quarto = "*"
 ptvsd = "*"
 
+[target.win-64.activation]
+scripts = ["env_setup.bat"]
+
 [target.linux-64.dependencies]
 quarto = "*"
 ptvsd = "*"
 
+[target.linux-64.activation]
+scripts = ["env_setup.sh"]
+
 [target.osx-64.dependencies]
 quarto = "*"
 ptvsd = "*"
+
+[target.osx-64.activation]
+scripts = ["env_setup.sh"]
+
+[system-requirements]
+linux = "3.10.0"
```

Comment:
This was needed for the linux build. We could leave it out of this PR, I don't know if there is anyone with a linux system that wants to test if it is working for ubuntu.